README.md

```diff
@@ -163,6 +164,8 @@ For any local testing do the following:
 ### Build Locally
 1. `docker build -t dsnp/instant-seal-node-with-deployed-schemas:latest -t dsnp/instant-seal-node-with-deployed-schemas:<versionNumberHere> .`
 
+Note: `--platform=linux/amd64` is required as `frequencychain` images are only published for the `linux/amd64` platform.
```

Comment:
This note also applies to `docker pull` in the previous section.

Do we want to say "only required if you're not already using linux/amd64" or just include it in the command line examples regardless?

Reply:
I was back and forth on it. I guess it still works if you are already on linux/amd64, so I'll update it all to have it.

Reply:
Updated